# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/tests/notifications/getnotifications-across-processes.https.window.js
# Commit: bda570180adf
# Full Hash: bda570180adfcc20891d5cfb47406756fe8903ad
# Author: Kagami Sascha Rosylight <saschanaz@outlook.com>
# Date: 2024-11-12 09:32:50
# Regressor Bug: 1928702
# File Overlap Count: 1
# Description:
#   Bug 1928702 - Part 11: Migrate ShowPersistentNotification to use IPC r=asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D227696
# ==============================================================================

diff -r c942992fa981 -r bda570180adf testing/web-platform/tests/notifications/getnotifications-across-processes.https.window.js
--- a/testing/web-platform/tests/notifications/getnotifications-across-processes.https.window.js	Mon Nov 11 20:34:15 2024 +0000
+++ b/testing/web-platform/tests/notifications/getnotifications-across-processes.https.window.js	Mon Nov 11 20:34:15 2024 +0000
@@ -1,6 +1,7 @@
 // META: script=/resources/testdriver.js
 // META: script=/resources/testdriver-vendor.js
 // META: script=/service-workers/service-worker/resources/test-helpers.sub.js
+// META: script=resources/helpers.js
 
 // (Cannot use `global=serviceworker` because testdriver only supports window)
 
@@ -14,8 +15,8 @@
   }
 });
 
-promise_setup(() => {
-  return test_driver.set_permission({ name: "notifications" }, "granted");
+promise_setup(async () => {
+  await trySettingPermission("granted");
 });
 
 service_worker_test("getnotifications-sw.js", "Service worker test setup");
